homeassistant/components/opower/manifest.json

```diff
@@ -6,5 +6,5 @@
   "dependencies": ["recorder"],
   "documentation": "https://www.home-assistant.io/integrations/opower",
   "iot_class": "cloud_polling",
-  "requirements": ["opower==0.0.20"]
+  "requirements": ["opower==0.0.21"]
```

Comment:
Reminder to bump this to the correct version once I merge and release https://github.com/tronikos/opower/pull/13